app/Models/Campaign.php

```diff
@@ -329,6 +329,9 @@ public function toSearchableArray()
         // Add boolean attribute for filtering website campaigns.
         $array['has_website'] = isset($this->contentful_campaign_id);
 
+        // Adds boolean attribute for filtering open/closed campaigns.
+        $array['is_evergreen'] = is_null($this->end_date);
```

Comment:
I wonder if we should configure these as computed properties (via `$this->is_evergreen` from a `getIsEvergreen` function) so that we could access the same logic elsewhere (e.g. a Customer.io or HTTP transformer) if we ever needed?

Reply:
Smarttttt! I can post a lil follow-up to move the pair of these computed ones to formal functions